forms-flow-api/src/formsflow_api/models/application_history.py

```diff
@@ -31,6 +34,9 @@ def create_from_dict(cls, application_audit_info: dict) -> ApplicationHistory:
             application_audit.submitted_by = application_audit_info["submitted_by"]
             application_audit.form_id = application_audit_info["form_id"]
             application_audit.submission_id = application_audit_info["submission_id"]
+            application_audit.color = application_audit_info.get("color")
+            application_audit.percentage = application_audit_info.get("percentage")
+            application_audit.is_resubmit = application_audit_info.get("is_resubmit")
```

Comment:
Doesn't the is_resubmit should be on the application model ? So that system can know from the application if the application is in re_submit state ?